### PR TITLE
45 lose screen

### PR DESCRIPTION
**Pull request by Sarvesh-Mk**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -97,6 +101,10 @@ def __show_help_screen(self, signal_emitter=None):
     def __show_prev_screen(self, signal_emitter=None):
         self.__set_overlay(self.prev)
 
+    def show_game_over_screen(self, signal_emitter=None):
```
Comment (yashkir) [inline main.py:104]: this should be private (__show_game_over_screen)

File: logic/StateManager.py [hunk 2]
```diff
@@ -33,41 +33,51 @@ def load_state(self, path: str) -> None:
...
     def set_state(self):
         """Sets the player state to the state that is loaded"""
-        cur_stats = self.player.stats
+        cur_stats = self.player_stats
         cur_stats.athletic_ability = self.state[0]['player']['stats']['athletic ability']
         cur_stats.charisma = self.state[0]['player']['stats']['charisma']
         cur_stats.smartness = self.state[0]['player']['stats']['smartness']
         cur_stats.wisdom = self.state[0]['player']['stats']['wisdom']
-        self.player.stats.update_text()
+        for x in self.state[0]['player']['stats']:
```
Comment (yashkir) [inline logic/StateManager.py:48]: what the hell is going on here?

**Review by yashkir — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (Sarvesh-Mk) [reply main.py:104]: im calling it from the state manager it has to be public

File: logic/StateManager.py (hunk 2 above)
Comment (Sarvesh-Mk) [reply logic/StateManager.py:48]: im loading the stats and i dont know how to make it with less code than that lol

Comment (yashkir) [reply logic/StateManager.py:48]: try harder

**Pull request closed without merging**